templates/Dataland/metadata.markup.twig

```diff
@@ -32,5 +32,14 @@
    "datePublished" : "{{dataset.acceptedDate|date('c', 'UTC') }}",
    "@type" : "Dataset",
    "@context" : "http://schema.org"
+   {% if boundingBox is not empty %},
+   "spatialCoverage": {
+      "@type": "Place",
+      "geo": {
+         "@type": "GeoShape",
+         "box": "{{boundingBox|default}}"
```

Comment:
The `|default` is not needed, you are checking it for `empty`, so `|default` would make it null is not set.